dayrepo/snippets/views.py

```diff
@@ -1,44 +1,56 @@
 from django.shortcuts import render
-from django.http import HttpResponse
+from django.http import HttpResponse, QueryDict
```

Comment:
```suggestion
from django.http import HttpResponse
```
こちらは結局使用しませんでしたので消しますmm